lighthouse-core/audits/audit.js

```diff
@@ -136,7 +137,18 @@ class Audit {
     if (displayValue === score) {
       displayValue = '';
     }
+    let auditDescription = audit.meta.description;
 
+    if((result.error || result.rawValue === false) && audit.meta.fallbackDescription) {
```

Comment:
With this, we only need one of these checks to set the `auditDescription`. Right? Why are there two here?